src/db/dbConnection.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
+import "dotenv/config";
+import bcrypt from "bcryptjs";
+import User from "../models/User.js";
+import Blogs from "../models/Blogs.js";
+
+async function connectDatabase() {
+    try {
+        await mongoose.connect(process.env.MONGODB_URL);
```

Comment:
Add the connection properties which were used before 
```
{
    useNewUrlParser: true,
    useUnifiedTopology: true,
 }
```

Reply:
they are soon to be deprecated npm gave me the warning and its running fine without it

Reply:
Okay then